curriculum/challenges/chinese/01-responsive-web-design/applied-visual-design/use-the-u-tag-to-underline-text.md

```diff
@@ -25,10 +23,8 @@ dashedName: use-the-u-tag-to-underline-text
 assert($('u').length === 1);
 ```
 
-`u` 标签的内容文本应为 `Ph.D. students`。
-
 ```js
-assert($('u').text() === 'Ph.D. students');
+
```

Comment:
```suggestion
assert($('u').text() === 'Ph.D. students');
```

Not sure why this disappeared

Reply:
Okay @RandellDawson this is a **weird** one.

So this is the file that we applied the custom segmentation rules to, which worked on the test project but not the curriculum project.

The two strings here that should be fixed by those segmentation rules disappear when I preview the translation download. If I remove the custom segmentation, then they are downloaded as expected. (I put the segmentation rules back, though).

Ideally, we would want those custom segmentation rules to **work** like they should (and do, on the test project). But that might be a Crowdin support thing. What are your thoughts on how to best fix this?

Reply:
So if we click the Reset button on the the custom segmentation screen and hide these strings, the downloaded version still excludes them?

Reply:
If we click the reset button on the segmentation, these strings are still hidden (because the segmentation isn't changing things like it should). And the downloaded version *does have* these strings as we expect.